src/mantine-core/src/Text/Text.styles.ts

```diff
@@ -96,6 +106,7 @@ export default createStyles(
       size,
       lineClamp,
       truncate,
+      truncateStart,
```

Comment:
Instead of adding another prop, change `truncate` prop signature to `truncate?: 'end' | 'start' | true`

Reply:
That's better! Thanks for the suggestion!